freshmaker/kojiservice.py

```diff
@@ -407,6 +407,41 @@ def get_bundle_csv(self, build_nvr):
         with zipfile.open(csv_files[0]) as f:
             return yaml.safe_load(f)
 
+    @region.cache_on_arguments()
+    def get_module_name_streams(self, build_nvr):
+        """
+        Get module's name and its streams through build NVR
+
+        :param str build_nvr: NVR of module build.
+        :return: a dict for module name and its streams, e.g. {"key": [values]}
+        :rtype: dict
+        """
+        module_name_streams_mapping = dict()
+        build = self.get_build(build_nvr)
+        modulemd_str = build.get("extra", {}).get("typeinfo", {}).get("module", {}).get("modulemd_str")
+        if modulemd_str:
+            try:
+                import gi
```

Comment:
I think there is no strong reason of importing the modules in method instead of the beginning of this module, right?

Reply:
Ack. Probably this is a bad habit from previous experience. We were annoying by cyclic issue, so for an unusual package and it is only used in one place, normally it will be put in lines.